reemgawad/aircams
# Design of summary page

What has changed
Put some design on the buttons
Put some color logic on the status display

How to test
Go to the summary page and make sure status gets updated when you approve/decline
The Status color should change accordingly ( orange/pending red/declined green/approved)
